### PR TITLE
feat(content-block-mixed): introducing content-block-mixed

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Description

Introducing web-components version

### Changelog

**New**

- Content Block Mixed storybook story and component;

### CODE REVIEW

**Review by asudoh — COMMENTED** [review]
Good progress so far @guilhermelMoraes! One comment - The story should be `.ts` instead of `.js`.

File: packages/web-components/src/components/content-block-mixed/__stories__/README.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import '../../content-block/content-block-paragraph';
+import '../../content-group/content-group-heading';
+import '../../content-group-cards/content-group-cards';
+import '../../content-group-cards/content-group-cards-item';
+import '../../content-group-pictograms/content-group-pictograms';
+import '../../content-group-simple/content-group-simple';
+import '../../content-item/content-item-heading';
+import '../../pictogram-item/pictogram-item';
+import '../../cta/card-cta';
+import '../../cta/card-cta-footer';
```
Comment (asudoh) [inline packages/web-components/src/components/content-block-mixed/__stories__/README.stories.mdx:28]: These `import`s should be from `@carbon/ibmdotcom-web-components` and  have `.js` extensions.

**Review by asudoh — APPROVED** [review]
LGTM, just one thing - Thanks @guilhermelMoraes!

**Comment by jeffchew** [comment]
Screen Shot 2020-11-30 at 9 30 08 AM

Hey @guilhermelMoraes , looks like there are some lint errors, can you fix?

**Comment by guilhermelMoraes** [comment]
@jeffchew Most of these errors are happening in files that I didn't change. Should I fix them here?

**Pull request merged**